public/controlpanel.php

```diff
@@ -27,10 +27,11 @@ function RenderUserPref(
     int $websitePrefs,
     int $userPref,
     bool $setIfTrue,
-    ?string $state = null
+    ?string $state = null,
+    ?int $targetLoadingIcon = 1
```

Comment:
I don't believe this is nullable

Reply:
You're right, this value should never be null. This was me getting the syntax for nullable and optional confused. Resolved in https://github.com/RetroAchievements/RAWeb/pull/1656/commits/c1a9d2e7df133b3896b5a27223795903087f6845.